### PR TITLE
[USETUP] Display the ReactOS release status

### PR DESCRIPTION
**Pull request by Fraizeraust**
## Purpose
As you guys know, ReactOS is in Alpha and barely stable. Having a notification concerning ReactOS' release status is critical as it informs the user the risks of running this operating system considering ReactOS is under heavy development and many bugs are still in the wild.

Displaying the said message in the 1st stage setup is a viable option.

## Presentation
Image URL: https://imgur.com/a/kpo0DtS

## TODO
As you can see, I have only edited the `en-US.h` file because the actual statements aren't that perfect. For this reason, I've set the Pull Request in WIP state as I request review from the devs to check if the wording is alright (or there needs some changes so it can fit better). Once everything is OK I'll edit the other language files.

- [x] Add the message page for other languages

### CODE REVIEW

**Comment by HBelusca** [comment]
Shouldn't this text be in the intro page instead?

**Comment by Fraizeraust** [comment]
That would create bloat in the Intro page, IMHO. I would rather want it to be in a separate page so that things can be clean and tidy.

Apart from this, what do you think about the statements from the text? Do you have a better suggestion or is it okay?

**Comment by ColinFinck** [comment]
If others also consider this disclaimer necessary, I would like you to replace the second page ("ReactOS Setup is in an early development phase") instead. The current text there is slowly getting outdated as we support more setup features and filesystems.

Adding yet another page would just bloat up the number of setup pages. Most people are already pushing RETURN all the time without reading the texts.

**Comment by gigaherz** [comment]
I personally don't mind having it (don't mind NOT having it either), but I do agree, the more pages we add, the higher the chances people go TL;DR and skip something important.

**Comment by ThFabba** [comment]
What Colin said. Please don't add more pages.

**Comment by HBelusca** [comment]
+1 for the idea of updating the existing disclaimer we already have.

**Comment by Fraizeraust** [comment]
As per the Colin's request, I will replace the second (ReactOS Setup) page to avoid pages bloat. Although this will take some time.

File: base/setup/usetup/usetup.c [hunk 1]
```diff
@@ -1088,20 +1088,19 @@ InstallIntroPage(PINPUT_RECORD Ir)
...
                 return QUIT_PAGE;
-
+            
             break;
         }
         else if (Ir->Event.KeyEvent.uChar.AsciiChar == 0x0D) /* ENTER */
         {
             return DEVICE_SETTINGS_PAGE;
-            // return SCSI_CONTROLLER_PAGE;
+            // return SCSI_CONTROLLER_PAGE
```
Comment (Fraizeraust) [inline base/setup/usetup/usetup.c]: Note to myself. I must add the semicolon back.

**Comment by Fraizeraust** [comment]
OK, I've replaced the second page as desired. However I still need approval upon the disclaimer text, if whether it is good and can fit or needs some changes.

**Comment by HBelusca** [comment]
Text is fine for me. But please revert the other unrelated changes (renamings of variables in the code, etc...)

File: base/setup/usetup/lang/en-US.h [hunk 2]
```diff
@@ -139,50 +139,55 @@ static MUI_ENTRY enUSIntroPageEntries[] =
...
-        TEXT_STYLE_NORMAL
+        "ReactOS Version Status",
+        TEXT_STYLE_HIGHLIGHT
     },
     {
         6,
-        9,
-        "support all the functions of a fully usable setup application.",
+        11,
+        "ReactOS is an Alpha operating system, meaning it is not feature-complete",
```
[the remark this answers is not in the document]
Comment (HBelusca) [reply base/setup/usetup/lang/en-US.h]: That would be better, yes.

**Comment by Fraizeraust** [comment]
@peterooch I already did.

The PR is no longer WIP, can be merged.

File: base/setup/usetup/lang/en-US.h [hunk 3]
```diff
@@ -139,50 +139,56 @@ static MUI_ENTRY enUSIntroPageEntries[] =
...
-        "support all the functions of a fully usable setup application.",
+        11,
+        "ReactOS is in Alpha stage, meaning it is not feature-complete",
         TEXT_STYLE_NORMAL
     },
     {
         6,
         12,
-        "The following limitations apply:",
+        "and under heavy development. It is recommended to use only for",
```
Comment (HBelusca) [inline base/setup/usetup/lang/en-US.h]: "It is recommended to use" *it* "only for..." <-- I think it's missing a "it" before "only" ?

File: base/setup/usetup/lang/en-US.h [hunk 4]
```diff
@@ -139,50 +139,56 @@ static MUI_ENTRY enUSIntroPageEntries[] =
...
+        "evaluation and testing purposes and not as your daily-usage OS.",
         TEXT_STYLE_NORMAL
     },
     {
-        8,
-        14,
-        "- File system checks are not implemented yet.",
+        6,
+        15,
+        "Backup your data or test on a secondary PC if you attempt",
```
Comment (HBelusca) [inline base/setup/usetup/lang/en-US.h]: s/PC/computer/

File: base/setup/usetup/lang/bg-BG.h [hunk 5]
```diff
@@ -139,59 +139,65 @@ static MUI_ENTRY bgBGWelcomePageEntries[] =
 
 static MUI_ENTRY bgBGIntroPageEntries[] =
 {
-    {
+   {
```
Comment (HBelusca) [inline base/setup/usetup/lang/bg-BG.h]: Indentation broken

File: base/setup/usetup/lang/fr-FR.h [hunk 6]
```diff
@@ -131,64 +131,64 @@ static MUI_ENTRY frFRWelcomePageEntries[] =
 
 static MUI_ENTRY frFRIntroPageEntries[] =
 {
-    {
+   {
```
Comment (HBelusca) [inline base/setup/usetup/lang/fr-FR.h]: Indentation broken

File: base/setup/usetup/lang/sv-SE.h [hunk 7]
```diff
@@ -137,7 +137,7 @@ static MUI_ENTRY svSEWelcomePageEntries[] =
 
 static MUI_ENTRY svSEIntroPageEntries[] =
 {
-    {
+   {
```
Comment (HBelusca) [inline base/setup/usetup/lang/sv-SE.h]: Indentation broken

**Comment by Fraizeraust** [comment]
@HBelusca Your reviews have been addressed.

File: base/setup/usetup/lang/en-US.h [hunk 8]
```diff
@@ -139,50 +139,56 @@ static MUI_ENTRY enUSIntroPageEntries[] =
...
-        "support all the functions of a fully usable setup application.",
+        11,
+        "ReactOS is in Alpha stage, meaning it is not feature-complete",
         TEXT_STYLE_NORMAL
     },
     {
         6,
         12,
-        "The following limitations apply:",
+        "and under heavy development. It is recommended to use it only for",
```
Comment (HBelusca) [inline base/setup/usetup/lang/en-US.h]: and *is* under ... (missing verb)

File: base/setup/usetup/lang/en-US.h [hunk 9]
```diff
@@ -139,50 +139,56 @@ static MUI_ENTRY enUSIntroPageEntries[] =
...
-        "- File system checks are not implemented yet.",
+        6,
+        15,
+        "Backup your data or test on a secondary computer if you attempt",
+        TEXT_STYLE_NORMAL
+    },
+    {
+        6,
+        16,
+        "to run on real hardware.",
```
Comment (HBelusca) [inline base/setup/usetup/lang/en-US.h]: ... to run *ReactOS* on real hardware.

**Comment by HBelusca** [comment]
After these I think this is good to go.

**Pull request merged**